custom/samveg/case_importer/validators.py

```diff
@@ -117,18 +127,14 @@ def run(cls, row_num, raw_row, fields_to_update, import_context):
         call_value, call_number = cls._get_latest_call_value_and_number(raw_row)
         if owner_name and call_number:
             if cls._upload_limit_reached(import_context, owner_name, call_number):
-                error_messages.append(
-                    _('Limit reached for {owner_name} for call {call_number}').format(
-                        owner_name=owner_name,
-                        call_number=call_number
-                    )
-                )
+                error_messages.append(UploadLimitReachedError())
```

Comment:
> It looks like the owner name is no longer displayed in the message, is that true?

Ya, I had to remove owner_name and call number.
Considering how the error is [added](https://github.com/dimagi/commcare-hq/blob/44f54201134f654a46ce05c6574b654600aad87e/corehq/apps/case_importer/do_import.py#L544), there is no real way to add information specific to a row in the error. 
The title of the error is used as the key. Then we have column_name and description. If we raise error with a message, that message is stored in the description but the description is updated each time a new error for the same key is added. So the description can be used if it stays the same for different errors, which I did for `RequiredValueValidator` in this [commit](https://github.com/dimagi/commcare-hq/pull/30921/commits/a52ab72473c0a851291225292ce948175b0a0892) by using mandatory columns instead of the missing values for the row.

Reply:
I think we are looking at roughly 3k rows in an upload, for now.

when it comes to the upload limit, I think the user is not going to pay much attention, at least for this use case here. The concern is that up to 40 records should be uploaded, and not which records were rejected because of upload.

If we need a separate row for each owner, we would need to make an exception class with no title and then pass the message with the owner name in it, which would lead to a separate section for each owner_name in the UI, which just leads to a overall longer error message in the UI.
So I had to decide between that or a compact message in the UI, where I chose the latter because the former won't be useful but if we are asked to make a separate row for each owner on limit, we can do so.